<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZKeyDLg</class>
 <widget class="QDialog" name="ZKeyDLg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="KeyNameLEd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Name</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="DuplicatesCHk">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Allow Duplicates</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBTn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisguardBTn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disguard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
